.changelog/4441.yml

```
@@ -0,0 +1,4 @@
+changes:
+- description: Split the testing of modeling rules on XSIAM to multiple machines
```

@samuelFain please take a look

---

This change log is currently not using one of the agreed [templates](https://confluence-dc.paloaltonetworks.com/display/DemistoContent/Changelogs).
I'd change it to something like:
```suggestion
- description: Modified the testing of modeling rules on XSIAM by splitting the process to multiple machines
```